.eslintrc.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "prettier/prettier": ["error", { endOfLine: "auto" }],
```

Comment:
Yeah It might be a good Idea when we have more to options to change for prettier. But in our case this was just added to ignore problems caused by different end of the line sequence in VS code. There won't be any other options to change, so I think we will just leave it as it is. Thanks for suggestion!